FlipperZero/.swiftlint.yml

```diff
@@ -5,11 +5,81 @@ excluded:
  - Frameworks
 
 disabled_rules:
- - force_try
  - identifier_name
- - line_length
- - force_cast
- - trailing_whitespace
- - opening_brace
+ - todo
+
+opt_in_rules:
+ - anyobject_protocol
```

Comment:
Why to disallow saying `class` instead of `AnyObject` with `anyobject_protocol`? I don't care, but is it really could be considered more readable?

Reply:
Because `class` is [deprecated](https://forums.swift.org/t/class-only-protocols-class-vs-anyobject/11507/10) in favor of `AnyObject`

Reply:
Didn’t know it is, it still didn’t produce a deprivation warning. Thx for info.